src/main/java/command/ByeCommand.java

```diff
@@ -0,0 +1,7 @@
+package command;
+
+public class ByeCommand extends Command {
+    public void execute(){
+        System.out.println("================================================");
```

Comment:
Printing the separation line is used several times in this program. Maybe you could place it as a variable?